packages/cxl-ui/scss/cxl-featured-course-card.scss

```diff
@@ -132,7 +132,7 @@
       @media #{mq.$large} {
         position: absolute;
         bottom: calc(-1 * var(--lumo-space-xl));
-        right: calc(2 * var(--lumo-space-xl));
+        right: calc(27.5% - 200px);
```

Comment:
This is still a lot of magic numbers :thinking: is there no way to have some canonical generic numberless positioning formula here?

Reply:
I like `left: calc(50% + 100px)` more. Easier to understand.

Looks good on `@media #{mq.$x-large}`
but doesn't look perfect `@media #{mq.$large}` (1024px) - too close to the right edge.

<img width="913" alt="Screenshot 2024-04-17 at 18 50 51" src="https://github.com/conversionxl/aybolit/assets/5165721/dc1b7927-3619-447a-b340-4a079cceef67">
<img width="728" alt="Screenshot 2024-04-17 at 18 51 00" src="https://github.com/conversionxl/aybolit/assets/5165721/3bb70b14-40a3-4825-81ab-f1c61135660f">